concrete/blocks/core_area_layout/form.php

```diff
@@ -160,53 +160,6 @@
 
                 },
                 <?php } ?>
-                cancelDesign(){
```

Comment:
This is kind of funny – it looks to me like these were ported over from the customizer in order to satisfy the way the old layout/customizer interaction worked. But those lines were actually not necessary – they were added years ago when we reworked how the layout toolbar worked, but then we reverted those changes without actually removing the call to the JS. So I'm removing these since they are not necessary but they do break interaction with the in-page editing on cancel.